source/backend/functions/settings/src/index.js

```diff
@@ -18,6 +18,19 @@ const ec2Client = new EC2({customUserAgent});
 const dbClient = AWSXRay.captureAWSv3Client(new DynamoDB({customUserAgent}));
 const docClient = DynamoDBDocument.from(dbClient);
 
+const ORG_AGGREGATOR_ERROR = 'The configuration aggregator cannot be created because your aggregator source type changed for that aggregator.'
```

Comment:
It might just be me but this error message sounds a bit weird because I'm not 100% clear what "that" is referring to here. It sounds like the configuration aggregator but then it's saying that hasn't been created

Reply:
This is the error that comes back from Config, when you try to update an Organization wide aggregator with fields that are specific only to a standard one it thinks you're trying to recreate it is as an incompatible type and throws this error.